src/App.js

```diff
@@ -1,23 +1,52 @@
-import logo from './logo.svg';
 import './App.css';
+import foods from './foods.json';
+import { useState } from 'react';
+import { SearchBar } from './components/SearchBar';
+import { AddFoodForm } from './components/AddFoodForm';
+import { List } from './components/List';
 
 function App() {
+  const [foodsList, setFoodsList] = useState(foods);
+  const [searchValue, setSearchValue] = useState('');
+
+  function handleSearch(event) {
+    setSearchValue(event.target.value);
+    console.log('Search Value: ', searchValue);
+  }
+  
+
+  //usually this works, but at times you have to click multiple times for the item to be deleted - in those cases it goes one parentNode too high -> the first console.log gives
+  //the UL element instead, so no food gets found there -> Index -1.
+  //But when clicking on the same button again one or two times it works, so not copmpletely sure why this error occurs in rare occasions. I guess having ".parentNode" 5times 
+  //isn't the most elegant way, perhaps it's due to that
+
+  function handleClickDelete (event) {
+    const nameOfElemToDelete =
+      event.target.parentNode.parentNode.parentNode.parentNode.parentNode.id;
```

Comment:
pass the id into the handleClickDelete instead of the event! That way things become much more elegant. Because you can use the id to do the filtering on the foods!